conda/meta.yaml

```diff
@@ -7,24 +7,27 @@ source:
     git_rev: v1.6.0
 
 build:
-    script: python setup.py install
+    number: 1
+    noarch: python
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
-  build:
-    - python {{ python }}
+  host:
+    - python
     - setuptools
+    - pip
 
```

Comment:
These belong in `host` rather than `build` according to conda-forge documentation.  `pip` is required for the preferred build command above.